### ISSUE TITLE
Listbox and Aurelia's templating syntax

### ISSUE THREAD

**Issue by domenkogler** [aurelia-ui-toolkits/aurelia-kendoui-bridge#788]
Is there any reason why aurelia templating syntax does not work inside Listbox?

**Comment by adriatic** [comment]
Yes there is: because Aurelia templating cannot cooperate successfully with KendoUI templating. There is no KendoUI API available for that.

**Comment by domenkogler** [comment]
Sorry, maybe my question was not formed good enough.
What is so special in Listbox? Other controls work perfectly.

**Comment by adriatic** [comment]
Similarly, my answer was too inclusive as the bridge does support Templates as shown **[here](https://aurelia-ui-toolkits-1.gitbook.io/kendoui-bridge-docs/bridge-developers-notes/template-compilation)**. Please provide a gist that illustrates your problem, and open the case to ensure the proper tracking of it (see **[Using gists and GistRun](https://aurelia-ui-toolkits-1.gitbook.io/kendoui-bridge-docs/troubleshooting/using-gists-and-gistrun)** for more details).

**Comment by domenkogler** [comment]
OK, this time I will be very specific. I am talking about this commented line:

That is line 18 of app.html of Listbox/templates in [this demo application](http://aurelia-ui-toolkits.github.io/demo-kendo/#/samples/listbox-templates) and also in [Gist](https://gist.run/?id=b826cc690bb0dd65a6ba9044aa0b2658).

**Comment by adriatic** [comment]
Let's ask @JeroenVinke for the answer to this, now _very specific_ question, as he wrote that comment. My guess would be that the Aurelia template compiler does not work sufficiently well with KendoUI template compiler in this context. 

@JeroenVinke, can you please address the issue phrased as

> What is so special in Listbox? Other controls work perfectly.

**Comment by JeroenVinke** [comment]
hey @domenkogler 

This has to be resolved by Telerik. The listbox control does not call an internal API like most other controls do, so our bridge can't do template compilation.

**Comment by domenkogler** [comment]
Thank you for the answer. I suspected there is something like this behind.
